### ISSUE TITLE
Default time stepping options might lead to rather excessive chopping.

### ISSUE THREAD

**Issue by blattms** [OPM/opm-simulators#1701]
I have  a faint memory that flow used to stop after excessive time step chopping. Has changed or should this still be the case?

I just run a test where the linear operator is causing trouble and the initial time step is chopped from 1 day down to 4.64115e-05. Seems like we allow very small time steps due to the default of  `MinTimeStepSize="0"`. I would guess that we will never drop below this threshold.

**Comment by GitPaean** [comment]
That is true. I think the current implementation limit the chopping times to be 10. It does produce very small time steps. 

On the other hand, there is some cases that the simulation can recover from very small time steps when things are difficult.

To tangle difficult convergence problem, ECL can suggest explicit solution, or current solution or shut the well has caused convergence problem.  We have not implemented anything related to the first two.  But when ECL begins suggesting explicit solution, the results do not look good very often. They just do not want to terminate the simulation for better use experience. It does not mean the result meaningful anymore.

**Comment by blattms** [comment]
Could not find any parameter to adapt this.

**Comment by GitPaean** [comment]
You are right. We should have some parameter to control them.
